### ISSUE TITLE
Move template to standalone repo?

### ISSUE THREAD

**Issue by jamesmontemagno** [CommunityToolkit/Maui.NativeLibraryInterop#42]
Could be cool if it was a template, would be easier to clone. Thoughts?

**Comment by vhugogarcia** [comment]
Just out of curiosity @jamesmontemagno. If it is created as a Github Repo Template, if the template gets updated, can we also update it from the origin, just like when we do a fork of a repo? I'm asking this because if the future if there are commit on the community toolkit side, the devs can also get them and update their fork.

Thanks in advance 😃

**Comment by kfrancis** [comment]
> Could be cool if it was a template, would be easier to clone. Thoughts?

Yes, and base it on the [plugin repo](https://github.com/jfversluis/Plugin.Maui.Feature).

**Comment by jamesmontemagno** [comment]
@vhugogarcia  yup, it is just a normal repo which is great ;) from my understanding.

**Comment by VladislavAntonyuk** [comment]
it would be even better to use it with `dotnet new Maui.NativeLibraryInterop -n MyLib`
